index.js

```diff
@@ -18,10 +20,16 @@ const m = module.exports = opts => {
 		return Promise.reject(new Error(`Couldn't find the file ${opts.filename}`));
 	}
 
-	const run = remark().use(remarkLint, config).process;
-	const file = toVfile.readSync(readmeFile);
+	return pkgConf('awesome-lint').then(configProject => {
+		return deepAssign(config, configProject);
```

Comment:
using `deep-assign` because the object structure `config.js` is more than one level deep